src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
```

Comment:
let's do semantics

Reply:
@huberciak, non-blocking comment:
- [`<main`](https://www.w3schools.com/tags/tag_main.asp) for a product card is not really suitable; it's more of a top-level content wrapper inside `<body`
- [`<article`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) will do better for a
> self-contained composition